stormevents/nhc/track.py

```diff
@@ -1235,6 +1235,52 @@ def correct_ofcl_based_on_carq_n_hollandb(
     :return: dictionary of forecasts for each advisory (aside from best track ``BEST``, which only has one hindcast) with corrected OFCL
     """
 
+    def clamp(n, minn, maxn):
+        return max(min(maxn, n), minn)
+
+    # Regression coefficients for the Rmax forecast
+    # ref: Penny et al. (2023). https://doi.org/10.1175/WAF-D-22-0209.1
+    def get_regression_coefs(fcst_hr, radii_values):
+        fhrs = [12, 24, 36, 48, 72, 96, 120]
+        regression_coefs = {
```

Comment:
It might make more sense for this dictionary to be defined as a global constant at the top of the file, or even in a separate file like `const.py` or `def.py`

Reply:
This would also make it easier to write test for the function alone without relying on hardcoded values in the test

Reply:
fair enough, yeah I'll try work on that.